### ISSUE TITLE
`import =` is not supported by @babel/plugin-transform-typescript

### ISSUE THREAD

**Issue by advancedtw** [stephenh/ts-proto#177]
it seems the pull request #150 is not fixing this issue.
more specific error message 
```
Please consider using `import <moduleName> from '<moduleName>';` alongside Typescript's --allowSyntheticDefaultImports option.
  269 |   } as any,
  270 | }
> 271 | import UnaryMethodDefinition = grpc.UnaryMethodDefinition;
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```

**Comment by advancedtw** [comment]
to make it compile I remove line 271 and add a line in the import statement like that 
```
/* eslint-disable */
import { BrowserHeaders } from "browser-headers";
import { grpc } from "@improbable-eng/grpc-web";
import { UnaryMethodDefinition } from "@improbable-eng/grpc-web/dist/typings/service";
import { Writer, Reader } from "protobufjs/minimal";
... 
```
I haven't tested in against a working web-grpc proxy yet I plan on doing that later today.

**Comment by stephenh** [comment]
Ah yeah, looks like this was a regression; it should be a quick fix, will take a look at it ~today-ish if I get a chance. Thanks for the report!

**Comment by advancedtw** [comment]
no problems and thanks a lot for keeping this project alive!

**Comment by stephenh** [comment]
Should be fixed in v1.41.1. Thanks!
